### ISSUE TITLE
Compiled TS files produces "No tests generated"

### ISSUE THREAD

**Issue by lvsanche** [Pythagora-io/pythagora#42]
Is there any way to transpile typescript so that pythagora can generate tests for it? 
I tried transpiling a simple addNumbers function 
```
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.addNumbers = void 0;
function addNumbers(a, b) {
    return a + b;
}
exports.addNumbers = addNumbers;
```

**Comment by LeonOstrez** [comment]
Hey @lvsanche thank you so much for testing Pythagora and opening the issue.

We just published new version of Pythagora 0.0.66 which should cover that. It would mean a lot if you could try it out again and let me know how it went. If there are some other issues please ping me and I will be happy to help.

**Comment by lvsanche** [comment]
thanks! Looks like my new obstacle to test is having a GPT-4 api key. I was trying to run this with a free trial personal API key

**Comment by LeonOstrez** [comment]
you can send me an email and I can create you Pythagora API key so that you can try it out. My email is leon.ostrez@gmail.com

Unfortunately, free trial doesn't give you access to GPT-4 and we are using only GPT-4

**Comment by lvsanche** [comment]
I received the API key and have been testing further with some transpiled TS.
Two things I've observed so far, (and let me know if I should open up separate issues for these)
<img width="1008" alt="Screen Shot 2023-06-20 at 2 13 08 PM" src="https://github.com/Pythagora-io/pythagora/assets/7565845/44042df2-278b-45b4-bcb8-1f281e279292">

From the screenshot, createDQ_1's relative import path appears to be getting inferred from AccountNotInterested.js's path and so I have to manually update it.

Also the import of accountNotInterestedQualifier is incorrect. The export in TS and in the transpiled JS is a named export. The edit is quick enough but just something that could be improved going forward. 
<img width="695" alt="Screen Shot 2023-06-20 at 2 16 57 PM" src="https://github.com/Pythagora-io/pythagora/assets/7565845/7631973a-8153-4019-8f09-00239a0c92fa">

**Comment by LeonOstrez** [comment]
hey @lvsanche today we released new version of Pythagora (pythagora@0.0.68) which should cover imports/requires much better. It would be great if you could try it out again and let us know how it works for you now.
